src/components/Dashboard/EventCard.vue

```diff
@@ -66,8 +66,11 @@ const eventDateLabel = computed(() => {
 			nextWeek: 'dddd',
 			sameElse: 'dddd'
 		})
-	// FIXME should be a translated string
-	return `${dateString} ${startDateString} - ${endDateString}`
+	return t('spreed', '{dateString} {startDateString} - {endDateString}', {
+		dateString,
+		startDateString,
+		endDateString,
+	})
```

Comment:
I feel like translators will have problems understanding this string...

Reply:
Maybe we can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/formatRange